content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md

```diff
@@ -33,10 +33,8 @@ To read more about {% data variables.product.prodname_dotcom %}'s privacy practi
 
 ## {% data variables.product.prodname_campus_program %} Application Eligibility
 
-* Often times, a campus CTO/CIO, Dean, Department Chair, or Technology Officer signs the terms of the program on behalf of the campus.
+* A school employee with authority to adopt technology for the whole school, such as a CTO/CIO, Dean, Department Chair, or Technology Officer, must sign the terms of the program on behalf of the campus.
```

Comment:
```suggestion
A school employee with authority to adopt technology for the whole school, such as a CTO/CIO, Dean, Department Chair, or Technology Officer, must sign the terms of the program on behalf of the campus.
```

Since we're deleting the second list item, this can just be a paragraph.